WORKSPACE

```diff
@@ -24,7 +24,8 @@ http_archive(
 load("@bazel_skylib//lib:versions.bzl", "versions")
 
 # Keep this version in sync with the BAZEL environment variable defined
-# in our .github/workflows/ci.yml config.
+# in our .github/workflows/ci.yml config, and the Bazel version on our
```

Comment:
Consider, instead:
"and the Bazel version installed for nightly builds (Googlers, see go/tb-tomservo)."

Also consider making it a list:

```
# Keep this version in sync with:
#  * The BAZEL environment variable defined in .github/workflows/ci.yml.
#  * The Bazel version installed for nightly builds (Googlers, see go/tb-tomservo).
```

Reply:
Done. Thanks!